layers/+distributions/spacemacs-base/keybindings.el

```diff
@@ -531,16 +531,17 @@
 (spacemacs|define-transient-state window-manipulation
   :title "Window Manipulation Transient State"
   :doc (concat "
- Select^^^^              Move^^^^              Split^^                Resize^^                     Other^^
- ──────^^^^───────────── ────^^^^───────────── ─────^^─────────────── ──────^^──────────────────── ─────^^──────────────────────────────
- [_j_/_k_] down/up       [_J_/_K_] down/up     [_s_] vertical         [_[_] shrink horizontally    [_q_] quit
- [_h_/_l_] left/right    [_H_/_L_] left/right  [_S_] vert & follow    [_]_] enlarge horizontally   [_u_] restore prev layout
- [_0_-_9_] window N      [_r_]^^   rotate fwd  [_v_] horizontal       [_{_] shrink vertically      [_U_] restore next layout
- [_w_]^^   other window  [_R_]^^   rotate bwd  [_V_] horiz & follow   [_}_] enlarge vertically     [_d_] close current
- [_o_]^^   other frame   ^^^^                  ^^                     ^^                           [_D_] close other"
+ Select^^^^               Move^^^^              Split^^               Resize^^             Other^^
+ ──────^^^^─────────────  ────^^^^────────────  ─────^^─────────────  ──────^^───────────  ─────^^──────────────────
+ [_j_/_k_]  down/up       [_J_/_K_] down/up     [_s_] vertical        [_[_] shrink horiz   [_u_] restore prev layout
+ [_h_/_l_]  left/right    [_H_/_L_] left/right  [_S_] verti & follow  [_]_] enlarge horiz  [_U_] restore next layout
+ [_0_.._9_] window 0..9   [_r_]^^   rotate fwd  [_v_] horizontal      [_{_] shrink verti   [_d_] close current
```

Comment:
I did a quick check through the <kbd>SPC ?</kbd> `transient`  (HELM Descbinds) entries.
#### Transient states
A comma `,` is used in these transient states:
- <kbd>SPC b .</kbd> Buffer Transient State
- <kbd>SPC l</kbd> Layouts Transient State
- <kbd>SPC l w</kbd> Workspaces Transient State

Two dots `..` are used in the <kbd>SPC n =</kbd> Evil Numbers Transient State  (+ or - instead of = also opens it).

Currently the <kbd>SPC w .</kbd> Window Manipulation Transient State uses a dash `-`, but if this PR is applied then it will use two dots `..`.

#### Documentation
In documentation.org these two lines use 2 dots `..`:

> [1..9, 0]	switch to nth layout
> [C-1..C-9, C-0]	switch to nth layout and keep the transient state active

https://github.com/syl20bnr/spacemacs/blob/develop/doc/DOCUMENTATION.org#layout-key-bindings

And news01.org uses 3 dots `...`:
>Spacemacs layouts now restrict the scope of the buffer list opened with SPC b b. Use SPC b B to list all the buffers of all the layouts. Also 1,2,...,9,0 keys now select a layout and close the layout menu, use C-1,C-2,...,C-9,C-0 to switch to a layout and keep the menu opened.

https://github.com/syl20bnr/spacemacs/blob/develop/news/news01.org#key-bindings
it's the next to last paragraph.

A dash `-` or a comma `,` won't work there. 

I found a page that talks about two vs three dots.
http://www.grammarphobia.com/blog/2008/12/two-dots-or-three.html
It states that two dots are used to save space, and the OED (Oxford English Dictionary) is a lot shorter because of it.

#### Question
Should `..` be used in all hydras (transient states)? Then it would be consistent in both the transient states and these two places in the news and documentation.

#### Notes
I have a Buffer TS formatting PR pending, but i didn't notice the comma as a sequence indicator when i made it. If two dots `..` should be used then i can update that PR, and make PRs for the layouts and workspaces as well.

About the two documentation.org lines. 
Is there a reason why they are written with the `0` and `C-0` after `1..9` and `C-1..C-9`? 
In the Layouts and Workspaces Transient States they are written starting from 0 up to 9: `[0,9]` and `[C-0,C-9]`. That seems more logical.